andela/kenya-cfh
# #153756406 user can create game

#### What does this PR do?
Enables User to start or create game

#### Description of Task to be completed?
>- Implement modal
>- save game logs to database after game end

#### How should this be manually tested?
* from terminal, run: `git clone https://github.com/andela/kenya-cfh.git` to clone repository
* cd into the cloned repository and run: `npm install` to install all required dependencies
* create a `.env` file in the root directory and set up Mongodb configurations as described in the `.env.sample` file.
*  run `npm test` to ensure all tests are passing
* run `npm run start:dev` to startup application and open browser at localhost:3000. 
* create an account
* play game with friends or strangers

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#153756406
#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-01-15 at 10 46 30 am" src="https://user-images.githubusercontent.com/28694656/34936395-7e240ca0-f9e1-11e7-891d-881045f3f246.png">
<img width="1436" alt="screen shot 2018-01-15 at 10 53 17 am" src="https://user-images.githubusercontent.com/28694656/34936601-5501ce9c-f9e2-11e7-82bc-b6dce4175b6c.png">
<img width="1440" alt="screen shot 2018-01-15 at 10 46 11 am" src="https://user-images.githubusercontent.com/28694656/34936408-89398944-f9e1-11e7-99f2-ddb66da54675.png">

#### Questions:
N/A
